crypto_kem/kyber1024/clean/verify.c

```diff
@@ -36,12 +35,12 @@ unsigned char PQCLEAN_KYBER1024_CLEAN_verify(const unsigned char *a, const unsig
 *              assumes two's complement representation of negative integers.
 *              Runs in constant time.
 *
-* Arguments:   unsigned char *r:       pointer to output byte array
-*              const unsigned char *x: pointer to input byte array
+* Arguments:   uint8_t *r:       pointer to output byte array
+*              const uint8_t *x: pointer to input byte array
 *              size_t len:             Amount of bytes to be copied
-*              unsigned char b:        Condition bit; has to be in {0,1}
+*              uint8_t b:        Condition bit; has to be in {0,1}
 **************************************************/
-void PQCLEAN_KYBER1024_CLEAN_cmov(unsigned char *r, const unsigned char *x, size_t len, unsigned char b) {
+void PQCLEAN_KYBER1024_CLEAN_cmov(uint8_t *r, const unsigned char *x, size_t len, unsigned char b) {
```

Comment:
Do you want to replace all the unsigned char? Here it's inconsistent.

Reply:
Need to grep a little bit harder still :p